sds_data_manager/utils/stackbuilder.py

```diff
@@ -21,7 +22,15 @@
 )
 
 
-def build_sds(scope: App, env: Environment, account_config: dict):
+def build_sds(
+    scope: App,
+    env: Environment,
+    account_config: dict,
+    rds_size: str = "SMALL",
+    rds_class: str = "BURSTABLE3",
+    rds_storage: int = 200,
+    use_custom_domain: bool = False,
```

Comment:
Could these be in the `account_config` dict? Maybe `rds_size = account_config.get("rds_size", "SMALL")` to get a default value inside here if you want.